src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Bau/BauCacheController.cs

```diff
@@ -147,6 +147,26 @@ public async Task<ActionResult> ClearPublicCachePublications()
             return NoContent();
         }
 
+        [HttpDelete("public-cache/publications/paths")]
+        public async Task<ActionResult> ClearPublicCachePublicationPaths(
+            ClearPublicCachePublicationPathsViewModel request)
+        {
+            if (request.Paths.Any())
+            {
+                var pathString = request.Paths.JoinToString('|');
+
+                await _publicBlobStorageService.DeleteBlobs(
+                    BlobContainers.PublicContent,
+                    options: new DeleteBlobsOptions
+                    {
+                        IncludeRegex = new Regex($"^publications/[^/]*/({pathString})$")
```

Comment:
Maybe we should be a bit more restrictive elsewhere and use `[^/]*` rather than `.*`.